pkg/didcomm/protocol/introduce/service.go

```diff
@@ -26,28 +31,173 @@ const (
 	AckMsgType = IntroduceSpec + "ack"
 )
 
+// message type to store data for eventing. This is retrieved during callback.
+type message struct {
```

Comment:
Can we rename this to something more meaningful? I know it's named that way in `didexchange.Service` - that is mistake.

Would this be better named as "EventMetadata"? Or simply "Event"?

Reply:
Done! `eventMetadata` bc it is kind of internal stuff